README.md

```diff
@@ -43,22 +43,29 @@ The Docker environment is recommended because of the fewer setup steps required
  * Install [git](https://git-scm.com/downloads)
  * Install [Git-LFS](https://git-lfs.github.com/)
 
+### Forking the repo 
+
+[Fork](https://github.com/odk-x/docs/fork) the odk-x/docs repository, so that a copy of odk-x/docs gets created in your github account.
+
 ### Cloning the repo
 
-Clone the docs repo into a directory you want the ODK-X docs files to be located. For example, navigate to the directory you want the files to be located using the "cd" (Change Directory) command on the command line:.
+Firstly, change to the directory(if there's one) you want to clone the ODK-X docs repository on your local machine.
```

Comment:
I think the original text was clearer here, suggest retaining.

Reply:
There could be ambiguity mentioning files , as the files will be in the docs, so you need to move to directory, where the docs directory will be located

Reply:
Correct, but the previous text also discussed navigating using cd. "Change to the director(if there's one)" is confusing and also the grammar of the sentence is not correct.

Reply:
okay,i will change it .

Reply:
made few changes, will you please review them